pkg/daemon/ceph/client/pool.go

```diff
@@ -263,6 +281,12 @@ func createReplicationCrushRule(context *clusterd.Context, clusterName string, n
 	return nil
 }
 
+func deleteCrushRule(context *clusterd.Context, clusterName string, ruleName string) {
+	args := []string{"osd", "crush", "rule", "rm", ruleName}
+	// ignore errors
+	_, _ = ExecuteCephCommand(context, clusterName, args)
```

Comment:
I think it would be appropriate to return the error let the caller (not) handle the error instead.